test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/TabViewSample.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using Tizen.NUI;
+using Tizen.NUI.Components;
+using Tizen.NUI.BaseComponents;
+
+namespace Tizen.NUI.Samples
+{
+    public class TabViewSampole : IExample
+    {
+        private TabView tabView;
+        private TabBar tabBar;
+        private TabButton[] tabButton;
+        private TabContent tabContent;
+        private View[] content;
+        private Button addBtn, removeBtn;
+        private int count;
+
+        public void Activate()
+        {
+            Window window = NUIApplication.GetDefaultWindow();
+
+            tabView = new TabView();
```

Comment:
When I set the size of a TabView to (200, 300), the content size looks good as what I expected but the tabs overflows the parent size.
Can you check what the problem is?
I ran the sample in my Ubuntu machine.